Dockerfile.nanoserver

```diff
@@ -0,0 +1,73 @@
+FROM microsoft/windowsservercore:1709 AS base
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV PYTHON_VERSION 3.6.5
+ENV PYTHON_HASH 9e96c934f5d16399f860812b4ac7002b
+ENV DESTINATION_FOLDER C:\\tools
+
+WORKDIR /tmp
+
+RUN $python_url = ('https://www.python.org/ftp/python/{0}/python-{0}-amd64.exe' -f $env:PYTHON_VERSION); \
+    Write-Host ('Downloading {0}...' -f $python_url); \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    (New-Object System.Net.WebClient).DownloadFile($python_url, '/tmp/python-installer.exe'); \
+    Write-Host ('Checking hash...' -f $python_url); \
+    if ((Get-FileHash python-installer.exe -Algorithm md5).Hash -ne $env:PYTHON_HASH) { Write-Host 'Python installer checksum verification failed!'; exit 1; }; \
+    $install_folder = Join-Path -Path $env:DESTINATION_FOLDER -ChildPath 'python'; \
+    Write-Host ('Installing into {0}...' -f $install_folder); \
+    Start-Process python-installer.exe -Wait -ArgumentList @('/quiet', 'InstallAllUsers=1', 'TargetDir={0}' -f $install_folder, 'PrependPath=1', 'Shortcuts=0', 'Include_doc=0','Include_pip=1', 'Include_test=0');
+
+WORKDIR /
+
+RUN Remove-Item tmp -Recurse -Force
```

Comment:
I find this cleanup in a separate stage to be a little strange.  I say that because the content that is being removed is already contained in a previous layer.  This isn't real important since this is an intermediate phase but you would never want to do this if the image from this layer were pushed/used.  A better pattern would be to delete the temp directory in the same layer in which it was created.

Reply:
Done